package.json

```diff
@@ -1,66 +1,23 @@
 {
   "name": "tala-speech",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "license": "GPL-3.0",
-  "homepage": "http://example.com",
-  "private": true,
-  "dependencies": {
-    "cors": "^2.8.5",
-    "express": "^4.18.1"
-  },
   "scripts": {
-    "start": "parcel src/index.tsx",
-    "build": "parcel build src/index.tsx --no-source-maps --dist-dir dist/",
-    "serve": "npx serve dist/ -l 80",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "author": "Talkamatic AB",
+  "packageManager": "yarn@4.0.2",
+  "dependencies": {
+    "speechstate": "https://git@github.com/vladmaraev/speechstate.git",
+    "typescript": "^5.3.3",
+    "typescript-language-server": "^4.3.1",
+    "vite": "^5.0.11",
+    "xstate": "^5.9.1"
```

Comment:
Just a note. Ideally, this should be using the version of xstate that speechstate is using.

Reply:
So should we downdate this to 5.5.1, or update speechstate to 5.9.1? 

I guess that updating ss to 5.9.1 would mean releasing 2.0.1 of ss.

Reply:
The latter. Didn't we release 2.0.0 of speechstate? I can't see it now.